pipeline/rating/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.16
+
+ARG VERSION=v4.2.0
+ARG BINARY=yq_linux_amd64
+
+WORKDIR /opt/core
+
+# https://github.com/helm/helm/releases
+# https://github.com/mikefarah/yq
+# https://github.com/team-soteria/rback
+RUN apk add jq bind-tools curl && \
+    wget https://github.com/mikefarah/yq/releases/download/$VERSION/$BINARY.tar.gz -O - |\
```

Comment:
see https://pkgs.alpinelinux.org/package/edge/community/x86/helm and https://pkgs.alpinelinux.org/package/edge/community/x86/yq, I guess we can just `apk add yq helm`

Reply:
https://github.com/alpine-docker/k8s just use this docker image. It has everything we need!